ui/src/components/Component-Login.vue

```diff
@@ -224,7 +195,26 @@ export default {
         // }
 
         this.isProcessing = true;
-        console.log({ accountid: this.accountID, password: this.password });
+        console.log({ accountid: this.accountID, email: this.email, password: this.password });
```

Comment:
You should not be logging the literal password to console, you should log 3 stars, "***", regardless of the length of the password.